### PR TITLE
Update README, by adding more references to installation options

### PR DESCRIPTION
**Pull request by EMaksy**
# Description

As in the description, adding a reference to the manual installation guide and the Ansible repository in the installation section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,14 @@ See [related documentation](./guides/alerting/alerting.md) for more information.
 
 **Trento** can be installed in various ways, depending on your needs and the requirements of its different components. Please refer to the specific [Agent](https://github.com/trento-project/agent) documentation for more information.
 
+## Installing Trento manually
+
+For manual instructions on installing Trento, please refer to our [Installation of Trento](https://github.com/trento-project/docs/blob/main/guides/manual-installation.md) guide.
+
+## Installing Trento using Ansible (WIP)
```
Comment (arbulu89) [inline README.md:78]: I would remove the `WIP` reference.
The ansible playbook is "finished" so to say.

File: README.md [hunk 2]
```diff
@@ -71,6 +71,14 @@ See [related documentation](./guides/alerting/alerting.md) for more information.
 
 **Trento** can be installed in various ways, depending on your needs and the requirements of its different components. Please refer to the specific [Agent](https://github.com/trento-project/agent) documentation for more information.
 
+## Installing Trento manually
```
Comment (arbulu89) [inline README.md:74]: I don't like the `manually` term.
What does it mean?
I would go simply with `Installing Trento`.
The same for the body of the chapter. `To install Trento, please refer to the

File: README.md [hunk 3]
```diff
@@ -71,6 +71,14 @@ See [related documentation](./guides/alerting/alerting.md) for more information.
...
+
+For manual instructions on installing Trento, please refer to our [Installation of Trento](https://github.com/trento-project/docs/blob/main/guides/manual-installation.md) guide.
```
Comment (arbulu89) [inline README.md:76]: `refer to our`, i would remove `our`. `please refer to the ... guide`

**Comment by EMaksy** [comment]
Thanks for the feedback, I adjusted and aligned all the installations options :+1:

**Review by abravosuse — APPROVED** [review]

**Review by arbulu89 — APPROVED** [review]

**Pull request merged**